src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  --nav-hover-color: #00ACDC;
+  --nav-color: black;
+  --header-bg-color: #fff;
+}
+
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: var(--header-bg-color);
+}
+
+.header-link {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: 60px;
+
+}
+
+.nav-list {
+  padding: 0;
   margin: 0;
+  display: flex;
+}
+
+.nav-item {
+  list-style: none;
+
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav-link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  line-height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--nav-color);
+}
+
+.nav-link:hover,
+.nav-link.is-active {
+  color: var(--nav-hover-color);
+}
+
+.nav-link.is-active::after {
+  position: absolute;
+  content: "";
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)